src/acceptance/app/dynamic_policy_test.go

```diff
@@ -254,6 +254,39 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 			WaitForNInstancesRunning(appGUID, 1, 10*time.Minute)
 		})
 	})
+
+	Context("when there is a scaling policy for cpuutil", func() {
+		BeforeEach(func() {
+			policy = GenerateDynamicScaleOutAndInPolicy(1, 2, "cpuutil", 40, 80)
+			initialInstanceCount = 1
+		})
+
+		It("should scale out and in", func() {
+			// this test depends on the size of the diego cells, currently 8 CPU 32 GB RAM,
+			// and the CPU entitlements per share configured in ci/operations/set-cpu-entitlement-per-share.yaml.
+			// if any of these dependencies change, the test may need some adjustments as well.
+			//
+			// the following shows the calculations this test is based on:
+			//   - diego cell = 8 CPU 32GB RAM
+			//   - total shares = 1024 * 32[gb host ram] / 8[upper limit of app memory] = 4096
+			//   - CPU entitlement per share = 8[number host CPUs] * 100/ 4096[total shares] = 0,1953
+			//   - CPU entitlement = 4096[total shares] / (32[gb host ram] * 1024) * (1[app memory in GB] * 1024) * 0,1953 ~= 25%
+			//
+			// in a nutshell: 1GB app memory results in a maximum cpu entitlement of 25%,
+			// this means that cpuutil will be 100% if app cpu is at 25%.
+
+			SetAppMemory(cfg, appName, "1GB")
+			cpuEntitlementOfAppWith1GBMemory := 25
```

Comment:
I think we should [make this configurable](https://github.com/cloudfoundry/app-autoscaler-release/tree/main/src/acceptance#test-configuration) as not everybody might have the same configuration and their acceptance would fail in that case.